cves/CVE-2019-6975.yml

```diff
@@ -49,7 +49,10 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
```

Comment:
Maybe make this a bit more high level, it's a difficult to understand the django technical language here.